cmd/create.go

```diff
@@ -0,0 +1,254 @@
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/orange-cloudavenue/cloudavenue-sdk-go/v1/infrapi"
+	"github.com/spf13/cobra"
+)
+
+var (
+	excrea1 = "#List all T0\ncav create vdc --name myvdc\n\n"
```

Comment:
- Improve name of variable
- Move example in example cobra struct

Reply:
Use CamelCase for the name of the variable and use the followinf syntax to write examples in multilines : 
 
```
nameOfTheVar = `
# xx
mycmd --flag
`
```